app-misc/hyfetch/hyfetch-9999.ebuild

```diff
@@ -40,6 +35,8 @@ python_install() {
 	cp neofetch "${D}/usr/bin/neowofetch" || die
 
 	rm -r "${D}/usr/lib/${EPYTHON}/site-packages/hyfetch/scripts" || die
```

Comment:
Can you fix this to use the functions from https://devmanual.gentoo.org/eclass-reference/python-utils-r1.eclass/index.html?

Reply:
I'll do that today. Apologies, had to be away from computer last week.

Reply:
No worries, thanks!